### PR TITLE
Honor bin.strpurge when printing esil strings

### PR DESCRIPTION
**Pull request by kazarmy**

### CODE REVIEW

File: libr/core/cbin.c [hunk 1]
```diff
@@ -199,7 +199,7 @@ static bool false_positive(const char *str) {
 	return false;
 }
 
-static bool strpurge(RCore *core, const char *str, ut64 refaddr) {
+R_API bool r_core_bin_strpurge(RCore *core, const char *str, ut64 refaddr) {
```
Comment (radare) [inline libr/core/cbin.c]: seems like something to be moved into RBin instead, but ill merge for now

Comment (kazarmy) [reply libr/core/cbin.c]: yep, the entire string filter subsystem apparently should be in `libr/bin/filter.c`

**Comment by radare** [comment]
Agree

**Comment by kazarmy** [comment]
Done in 6484c12547b548232c03243c2dbf529b58eb440c.

**Pull request merged**